File: clang/UKL/soal1.c
/*
Soal        : 1
A           : 5
B           : 9
Suku akhir  : 21
*/

#include <stdio.h>

int main(){
    int A=5,
        B=9,
        suku_akhir=2,
        u=A,
        s=A;
    printf("Deret Aritmatikanya adalah \n");
    for(int i=0;i<suku_akhir;i++){
        printf("Angka U awal = %d\n", u);
        printf("Angka S awal = %d\n", s); 
        u=u+B;
        s=s+u;   
        printf("Angka U awal = %d\n", u);
        printf("Angka S awal = %d\n", s); 
    }
    printf("S = s - u = %d - %d = %d\n", s, u, (s-u));
    s=s-u;
    printf("Jumlah Deret Aritmatikanya adalah %d\n", s);
}

File: X/typescript/soal1.ts
/*
Soal        : 1
a           : 5
b           : 9
Suku akhir  : 21
*/

let a = 5,
  b = 9,
  suku_akhir = 2,
  u = a,
  s = a;

console.log("Deret aritmatikanya adalah");

for (let i = 0; i < suku_akhir; i++) {
  console.log("Angka U awal = " + u);
  console.log("Angka S awal = " + s);
  u += b;
  s += u;
  console.log("Angka U akhir = " + u);
  console.log("Angka S akhir = " + s);
}

console.log("S = s-u = " + s + " - " + u + " = " + (s - u));
let total = s - u;
console.log("Jumlah Deret aritmatikanya adalah " + s);
